packages/platform-sdk-btc-indexer/src/types.ts

```diff
@@ -13,13 +13,13 @@ export interface Flags {
 	batchSize: number;
 }
 
-export interface VOut {
+export interface Output {
```

Comment:
Nice, feels less like cpp code.

Found this [here](https://learnmeabitcoin.com/technical/vout): The “v” in vout stands for vector, because outputs in the bitcoin source code are stored in a vector data structure (which is basically an array)